gratipay/models/team/mixins/takes.py

```diff
@@ -71,19 +72,23 @@ def set_take_for(self, participant, take, recorder, cursor=None):
 
         """
         def vet(p):
-            assert not p.is_suspicious, p.id
-            assert p.is_claimed, p.id
-            assert p.email_address, p.id
-            assert p.has_verified_identity, p.id
+            if p.is_suspicious:
+                raise NotAllowed(_("user must not be flagged as suspicious"))
```

Comment:
That way, you'd only hit this if you were hitting the API directly. At that point I think English error messages are okay.